pkg/network/tracer/tracer.go

```diff
@@ -242,8 +243,12 @@ func newConntracker(cfg *config.Config, bpfTelemetry *nettelemetry.EBPFTelemetry
 
 	var c netlink.Conntracker
 	var err error
-	if c, err = NewEBPFConntracker(cfg, bpfTelemetry); err == nil {
-		return c, nil
+	// try creating ebpf conntracker 3 times in case the module is not loaded on the host yet
+	for i := 0; i < 3; i++ {
```

Comment:
You can use `github.com/avast/retry-go/v4` to do this "more cleanly" if you want

Reply:
(and it's already in use by CWS and `pkg/util/kernel` so no additional dependency

Reply:
thanks! I will try this

Reply:
```	
if err = retry.Do(
		func() error {
			c, err = NewEBPFConntracker(cfg, bpfTelemetry)
			return err
		},
		retry.Attempts(3),
		retry.Delay(1*time.Second),
	); err == nil {
		return c, nil
	}
```
this is what it looks like using the retry library... I think it may be clearer what's happening with just the for loop? what do you think?

Reply:
I would use the `retry` library. I think if you remove the `if` around the `retry.Do` it may look cleaner (move the `if` after).